recipes/glib/all/test_package/conanfile.py

```diff
@@ -4,7 +4,7 @@
 
 class TestPackageConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake", "cmake_find_package"
+    generators = "cmake", "cmake_find_package", "pkg_config"
```

Comment:
Why adding pkg_config generator in test_package? No pkg-config file is consumed afterwards in CMakeLists.

Reply:
it makes sure .pc files can be generated. There was some cases where I realized that the package_info method was wrong because the pkg_config generator crashed.